settingtypes.txt

```diff
@@ -3,6 +3,9 @@
 # Static paths do not work well with settings
 #areas.filename (Configuration file path) string (world_path)/areas.dat
 
+# Use smallest area volume precedence concept. (experimental may change)
```

Comment:
Is this descriptive enough? How about something along these lines?
```
Determine area access based on the smallest area volume that contains the interaction position.
If set to `false`: interacting is allowed if there is any open or shared area that includes the interaction position.
```

Reply:
This is what I've come up with so far. Does it convey clearly enough, that when set to true, areas within areas can't restrict but only open more?

```
# Use smallest area volume precedence concept. (experimental may change)
# Determine area access based on the smallest area volume that contains
# the interaction position. Sharing areas is achieved with factions instead
# of using /add_owner.
# This allows players to have private areas within a greater open/shared area
# and also define open/shared areas within those private areas.
# If set to `false`: interacting is allowed if the interaction position
# resides in the players own area, area is open or shared with factions.
# Even if other more restrictive areas enclose the interaction position.
```

The text is already rather long so I didn't want to repeat in other words. However I do think it is important for users to understand this difference. It is what drove me to request this addition.

Reply:
That does indeed explain it much in detail. However, I'm under the impression that it's grammatically not entirely correct. Also adding some structure might make it easier to comprehend:
```
# Use smallest area volume precedence concept. (experimental; may change)
#
# If set to `true`: 
#    The interaction permission is defined by the smallest area volume that
#    contains the interaction position. Granting access to areas is achieved
#    by factions instead of using `/add_owner`.
#    This allows players to have private areas within a greater open/shared
#    area and also define open/shared areas within those private areas.
# If set to `false`: (default)
#    Interacting is permitted if the interaction position resides in any of the
#    player's own areas, shared or open areas.
#    This permission is not impacted by more restrictive, intersecting areas.
```
What do you think?

Reply:
structure is good, yes. Will do that.